stage1/prepare-app/prepare-app.c

```diff
@@ -396,6 +436,11 @@ int main(int argc, char *argv[])
 	pexit_if(symlink("/dev/pts/ptmx", to) == -1 && errno != EEXIST,
 		"Failed to create /dev/ptmx symlink");
 
+	/* Move symlinks to device node volumes to "/dev/.rkt" so they can be
+	 * used in the DeviceAllow= option of the app's unit file (systemd
+	 * needs the path to start with "/dev". */
+	move_volume_symlinks();
```

Comment:
This seems slightly misleading to me, as it is not really moving anything around, just adding another layer of symlinks? Has the implementation been changed in the meanwhile?